.evergreen.yml

```diff
@@ -82,39 +89,35 @@ functions:
         env:
           KUBECONFIG: ${workdir}/kube_config
 
-  run_e2e_test:
-    - command: subprocess.exec
+  black_formatting_test:
+    - command: shell.exec
       type: test
       params:
-        add_to_path:
-          - ${workdir}/bin
         working_dir: mongodb-kubernetes-operator
-        env:
-          KUBECONFIG: ${workdir}/kube_config
-        command: "pip3 install -r ./requirements.txt"
+        script: |
+            . venv/bin/activate
+            pip3 install black
+            black --check `find . -type f -not -path '*venv*' -name '*.py'`
 
-    - command: subprocess.exec
+  run_e2e_test:
+    - command: shell.exec
       type: test
       params:
         working_dir: mongodb-kubernetes-operator
-        add_to_path:
-          - ${workdir}/bin
-        env:
-          KUBECONFIG: ${workdir}/kube_config
-        command: "python3 ./scripts/dev/e2e.py --test ${test} --tag ${version_id} --config_file ./scripts/ci/config.json"
+        script: |
+            . venv/bin/activate
```

Comment:
I think generally we want to try and avoid using `script` as it can get quite out of hand in general.

We could turn it into a small bash script in `scripts/ci`

and use the `binary` option instead of `script`